docs/actions.md

```diff
@@ -286,6 +286,26 @@ https://${APIHOST}/api/v1/namespaces/${NAMESPACE}/actions/actionName
 action invocations using the REST interface, see
 [Using REST APIs with OpenWhisk](rest_api.md#actions).
 
+### Saving action code
+
+Code associated with an existing action can be fetched and saved locally. Saving can be performed on all actions except sequences and docker actions. When saving action code to file, the code will be saved in the current working directory, and the saved file path will be displayed.
+
+1. Save action code to a filename that corresponds with an existing action name. A file extension that corresponds to the action kind will be used, or an extension of .zip will be used for action code that is a zip file.
+  ```
```

Comment:
Change:
A file extension that corresponds to the action kind will be used, or an extension of .zip will be used for action code that is a zip file.
To:
A file extension that corresponds to the action kind is used, or an extension of `.zip` will be used for action code that is a zip file.